steps/previsit/first-attempt.tsx

```diff
@@ -0,0 +1,129 @@
+import formatDate from "date-fns/format";
+import { Heading } from "lbh-frontend-react";
+import {
+  FieldsetLegend,
+  HeadingLevels,
+  Paragraph
+} from "lbh-frontend-react/components";
+import React from "react";
+import {
+  ComponentDatabaseMap,
+  ComponentWrapper,
+  DynamicComponent,
+  StaticComponent
+} from "remultiform/component-wrapper";
+import { Checkboxes, CheckboxesProps } from "../../components/Checkboxes";
+import { makeSubmit } from "../../components/makeSubmit";
+import { TextArea } from "../../components/TextArea";
+import ProcessDatabaseSchema from "../../storage/ProcessDatabaseSchema";
+import processRef from "../../storage/processRef";
+import PageSlugs from "../PageSlugs";
+import PageTitles from "../PageTitles";
+
+const step = {
+  title: PageTitles.FirstAttempt,
+  heading: "Unable to enter the property",
+  step: {
+    slug: PageSlugs.FirstAttempt,
+    nextSlug: PageSlugs.Loading,
+    submit: (nextSlug?: string): ReturnType<typeof makeSubmit> =>
+      makeSubmit([
+        {
+          slug: nextSlug as PageSlugs | undefined,
+          value: "Save and continue"
+        },
+        {
+          slug: PageSlugs.Loading,
+          value: "Cancel"
+        }
+      ]),
+    componentWrappers: [
+      ComponentWrapper.wrapStatic(
+        new StaticComponent({
+          key: "first-attempt-heading",
+          Component: Paragraph,
+          props: {
+            children: (
+              <Paragraph>
+                <Heading level={HeadingLevels.H2}>
+                  First Attempt: {formatDate(new Date(), "d MMMM yyyy")}
```

Comment:
We'll need to do something to persist this date, though. I think we probably want to do something special when clicking the unable to enter button to save the date it was pressed, so reloads of this page don't change the date, and then read it from the store. Not required for this PR, though.